AutoRest/Generators/Ruby/Azure.Ruby/Templates/AzureMethodTemplate.cshtml

```diff
@@ -27,29 +26,19 @@ else if (Model.HttpMethod == HttpMethod.Post || Model.HttpMethod == HttpMethod.D
 {
 @:@WrapComment("# ", string.Format("@param {0} {1}{2}", parameter.Name, parameter.Type.GetYardDocumentation(), parameter.Documentation))
 }
-@WrapComment("# ", "@return [Concurrent::Promise] promise which provides async access to http response.")
+@WrapComment("# ", string.Format("@param custom_headers [{0}] A hash of custom headers that will be added to the HTTP request.", "Hash{String => String}"))
 #

```

Comment:
@vishrutshah They weren't in local parameters and looking into [MethodTemplate](https://github.com/Azure/autorest/blob/master/src/generator/AutoRest.Ruby/Templates/MethodTemplate.cshtml), I saw that they were added in the template code.